src/main/java/club/sk1er/hytilities/config/HytilitiesConfig.java

```diff
@@ -209,6 +209,14 @@
     )
     public static boolean hidePlayerRanksInTab;
 
+    @Property(
+        type = PropertyType.SELECTOR, name = "Highlight Friends In Tab",
+        description = "Add a star to the names of your Hypixel friends in tab.",
+        options = {"Off", "Left of Name", "Right of Name"},
```

Comment:
move the options line to be the last in the annotation to follow the rest of how this class looks